governance/polls/Sky Protocol Launch Season - Token and Product Launch Parameter Proposal - September 9, 2024.md

```diff
@@ -0,0 +1,133 @@
+* Corresponding [Atlas V2](https://sky-atlas.powerhouse.io/) Active Data Updates
+
+To learn more about Launch Season, refer to:
+
+* [MakerDAO Endgame: Launch Season (final)](https://forum.makerdao.com/t/makerdao-endgame-launch-season-final/24740)
+* [Sky has arrived!](https://forum.makerdao.com/t/sky-has-arrived/24959)
+* [Atlas V2](https://forum.makerdao.com/t/atlas-v2-upgrade-poll-request/25010)
+* [Endgame: Announcing the Token and Product Launch forum post](https://forum.makerdao.com/t/endgame-announcing-the-token-and-product-launch/25021/1)
+
+### Launch Executive Proposal
+
+* Set the Schedule Method time for the earliest spell execution: September 17, 12:00 UTC
+
+### Deployment Requirements Before Token and Product Launch Spell
+
+* Tokens:
+  * USDS
+  * SKY
+  * sUSDS
+* DAI:USDS and MKR:SKY converters
+* Create USDS/SKY UniV2 pool (if this does not exist already)
+* FlapperUniV2 for the Smart Burn Engine
+* Splitter
+* SplitterMOM
+* LitePSM USDS Wrapper
+* USDSJoin
+* SKY DssVestMintable
+* SKY VestingRewardsDistribution
+* Rewards distribution cron-job
+* StakingRewards (rewardsToken: SKY, stakingToken: USDS)
+* StakingRewards (rewardsToken: 01, stakingToken: USDS)
+
+### Parameters
+
+* New Tokens Initialization
+  * [USDS](https://github.com/makerdao/nst)
+  * [SKY](https://github.com/makerdao/ngt)
+  * [Savings USDS (sUSDS)](https://github.com/makerdao/sdai/tree/susds)
+    * Set Sky Savings Rate to **6.25%**
+* Initialize token converters
+  * DAI <> USDS converter
+    * DAI to USDS conversion (both ways). The exchange Rate is **1:1**
+  * MKR <> SKY converter
+    * MKR to SKY conversion with an MKR/SKY conversion rate of **1:24,000**
+* USDS/SKY UniV2 Pool Migration and Smart Burn Engine Upgrade to use New Tokens
+  * USDS/SKY UniV2 Pool Migration
+    * Migrate funds from DAI/MKR UniV2 Pool to USDS/SKY UniV2 Pool
+  * Splitter Initialization:
+    * `hump`: **55 million DAI**
+    * `bump`: **65,000 DAI** (USDS in practice)
+    * `burn`: **100%** (1.0 * WAD)
+    * `hop`: **10,249 seconds**
+  * Dss Flapper Initialization to Use New Tokens
+    * `flapper`: FlapperUniV2
+    * `pip`: OracleWrapper ([0x38e8c1D443f546Dc014D7756ec63116161CB7B25](https://etherscan.io/address/0x38e8c1D443f546Dc014D7756ec63116161CB7B25))
+    * `want`: **0.98 * WAD**
+* Sky Token Rewards (STR) Deployment
+  * SKY (Supply USDS to get Sky Token Rewards in the form of SKY)
+    * DssVest
+      * Rewards Distribution Rate: **600,000,000 SKY**
+      * Rewards Distribution Cap: **800,000,000 SKY**
+      * Start Date relative to the spell cast time minus 7 days
+        * `vestBgn`: `block.timestamp - 7 days`
+      * End Date is set one year after Start Date defined by duration
+        * `vestTau`: **365 days**
+      * Call `distrdibute()` in VestedRewardsDistribution contract in the spell execution
```

Comment:
Should this be `distribute()` ?

Reply:
fixed in 3806c4d18ffbcb53765d683b0b79f0e3913c3c8a